internetarchive/openlibrary
# Add stale reading log data to solr

Closes #7321 . Feature. Adds fields to solr to keep track of reading log counts.

### Technical
Note these are stale because they do not auto-update when someone adds/removes books from their reading logs. That will be done separately. For now these numbers will only update on full reindex, or when a work is edited.

### Testing
Tested locally:
- Remove volumes relating to solr
- `docker-compose up -d`
- Wait for home node to finish
- [x] Confirm reading log data in solr admin dashboard

solrbuilder:
- [x] Run solrbuilder, confirm reading log data in solr admin dashboard.

### Screenshot
![image](https://user-images.githubusercontent.com/6251786/221301073-b97db9b8-4241-4966-beb1-95c3e5ff1cfb.png)

### Stakeholders
